ex2.sh

```diff
@@ -0,0 +1,29 @@
+# q1:
+ls -S
+
+# a : cat /etc/services
+
+# b:ls -S /etc/ | head -1
+    cat /etc/services
+
+# q2:
+ls -S -r
+
+# a:ls -S /etc/ | tail -1
+    cat /etc/motd
+
+# q3:
+
+mkdir -p grandpa/father/son
+
+
+# q4:
+bc
+
+# q5:
+bc -q
+
+# q6:
+cal 5 2023
+
```

Comment:
?? 6a , b

Reply:
This exercise have just one section

Reply:
I checked it now and was just 6a

Reply:
Maybe should be ncal so this will display all the months of the year